### ISSUE TITLE
High GPU usage

### ISSUE THREAD

**Issue by Symbai** [dotnet/wpf#6082]
* .NET Core Version: 3.1 and 6.0
* Windows version: Windows 10 and Windows 11
* Does the bug reproduce also in WPF for .NET Framework 4.8?: I dont know, all apps are only .NET (Core)
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? No
 
 **Problem description:**
For some people my applications use like 20-40% of GPU usage, for some even 90% when they're visible. The same applications work pretty fine for me and a lot of other people. When minimized to taskbar the GPU usage goes down to 0%. These people have extremely strong graphic cards like Nvidia 3080. My applications don't do any special that would explain the high GPU usage and I don't use any third party controls.
 
 **Actual behavior:** 
High GPU usage
 
 **Expected behavior:**
Low GPU usage 

 **Minimal repro:**
As this is per computer a "minimal repro" wouldn't help here.

**Comment by lindexi** [comment]
@Symbai Can I know what happened to your app?

**Comment by gurpreet-wpf** [comment]
@Symbai Have you tried any other DirectX based application on the same machine?

**Comment by Symbai** [comment]
Yes all non-WPF applications run fine. But all WPF applications instead have a high GPU usage under the given circumstances.

**Comment by ClosetBugSlayer** [comment]
We've known for a decade that WPF does not tesselate circles (and possibly rounded edges too) efficiently.
